webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -324,7 +324,7 @@ class ResourcesResponderV2 extends ResponderWithStandoffV2 {
                 resourceClassIri = internalCreateResource.resourceClassIri.toString,
                 resourceLabel = internalCreateResource.label
             ),
-            values = valuesWithValidatedPermissions
+            values = sparqlForValuesResponse.unverifiedValues
```

Comment:
This was doing the update verification with the (escaped) values that were sent to `ValuesResponderV2`, not with the `UnverifiedValueV2` objects returned by `ValuesResponderV2`, which should have been (but weren't) unescaped.